lib/metaschema-config/category.js

```diff
@@ -58,6 +58,16 @@ const addCategoryData = (type, entity, ms) => {
   return errors;
 };
 
+const checkDetail = category =>
+  ['Catelog', 'Subdivision', 'Hierarchy']
+    .filter(type => category[type.toLowerCase()])
+    .map(
+      type =>
+        new SchemaValidationError('detailHierarchy', category.name, {
```

Comment:
Could we add to the error message what category the detail comes from to be clear?
(`Category '${source}' cannot be divided by ${type} while being a detail in '${categoryName}.${fieldName}`)
(`Category '${source}' cannot be divided by ${type} while being a detail (in '${categoryName}.${fieldName})`)